### PR TITLE
Fixes layer highlighting/map interaction bug.

### PR DESCRIPTION
**Pull request by tomauer**
Important bug fix.

### CODE REVIEW

File: app/js/mol.map.layers.js [hunk 1]
```diff
@@ -554,16 +554,20 @@ mol.modules.map.layers = function(mol) {
                     
                     l.layer.click(
                         function(event) {
+                            var boo = false,
```
Comment (eightysteele) [inline app/js/mol.map.layers.js:558]: Nice var name. :)

**Comment by eightysteele** [comment]
LGTM. #shipit

**Comment by tomauer** [comment]
One more tweak...just a second.

**Comment by tomauer** [comment]
Ok. Now we're good to go. Merging.

**Pull request merged**